src/css/base.scss

```diff
@@ -257,6 +257,13 @@ a:focus {
     width: 50%;
 }
 
+.w-49 {
```

Comment:
Are `w-49`& `w-51` generic css as we are creating it in base.scss ?

Reply:
Removed from base